alarm/__init__.py

```diff
@@ -1 +1,3 @@
-__version__ = '0.1.0'
+from importlib.metadata import version
+
+__version__ = version(__name__)
```

Comment:
just learned something new ;)

Reply:
got it from Karmabot actually :)

Reply:
I am changing this to the flit way:

```
"""A package to play an alarm after an interval"""

__version__ = "0.0.1"
```

then in toml:

```
[project]
...
dynamic = ["version", "description"]
``